pkg/cvo/availableupdates.go

```diff
@@ -53,15 +52,15 @@ type availableUpdates struct {
 
 	At time.Time
 
-	Updates   []cvv1.Update
-	Condition osv1.ClusterOperatorStatusCondition
+	Updates   []configv1.Update
```

Comment:
this `configv1.Update`  is little ambiguous.

Reply:
Good point - I realized last night too that when we start bringing metadata back from cincinatti (messages and URLs to display pages) we're going to need to split the update.  I was going to suggest `ClusterVersionUpdate` and `AvailableClusterVersionUpdate` as the two structs.